MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorPressedToolbarTextColor             
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text when it is pressed.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorDisabledToolbarTextColor              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text when it is disabled.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipBackground               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipTextColor                
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip text.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorFocusedColorPaletteColor                 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of color palate icon when it is selected.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorSelectionStroke                  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of crop layer and annotations selection stroke.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalSavePopupBackground                   
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of saved message popup background.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalSavePopupTextColor                    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of saved message popup text.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorToolbarIconSeparatorColor                     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tool bar icons divider line.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalToolbarTextFontSize                      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of toolbar text.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipTextFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of tooltip text.      
```

Comment:
Font size on tooltip text in image editor

Reply:
Updated